core/src/test/java/apoc/export/csv/ExportCsvUseTypeTest.java

```diff
@@ -0,0 +1,80 @@
+package apoc.export.csv;
```

Comment:
Because there are extra nodes that would change the results of the ExportCsvTest test, so I created a new file to not make a lot of changes in that file and to simplify the assertions